query-service-impl/src/test/resources/application.conf

```diff
@@ -231,4 +231,4 @@ service.config = {
       }
     }
   ]
-}
\ No newline at end of file
+}
```

Comment:
I guess no change here.

Reply:
New line got added. Reverted that changed.